server/sonar-server/src/main/java/org/sonar/server/user/UserUpdater.java

```diff
@@ -211,10 +214,10 @@ private static void checkNotEmptyParam(@Nullable String value, String param, Lis
   private static void validateLoginFormat(@Nullable String login, List<Message> messages) {
     checkNotEmptyParam(login, LOGIN_PARAM, messages);
     if (!Strings.isNullOrEmpty(login)) {
```

Comment:
you already check that the login is null or empty. It would be more logical to check that you have no message. WDYT ?